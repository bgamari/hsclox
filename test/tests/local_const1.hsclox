{
    var b = 4;
    b = 3 + b;
    const a = 1;
    a = 2;
    print a;
}
