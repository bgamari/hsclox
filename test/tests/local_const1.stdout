[line 5] Error at '=': Can't reassign const variable
